ingredients.js

```diff
@@ -0,0 +1,81 @@
+const ingredientsSet = new Set()
+// ingredients = localStorage.getItem('ingredients')
+
+// ingredients.add(setIng)
+const createSpanBtns = () => {
+    const $pencilIcon = $('<i>').addClass('fa-solid fa-pencil')
+    const $trashIcon = $('<i>').addClass('fa-solid fa-trash-can')
+    
+    $trashIcon.click(event => {
+        removeIngredient(event)
+        $(event.target).parentsUntil($('.content__list')).remove()
+      })
+
+    $pencilIcon.click(event => {
+        editIngredient(event)
+    })
+    
+    return $('<span>').append($pencilIcon, $trashIcon)
+}
+
+export const getLocalStorageData = () => {
+    if (localStorage.getItem('ingredients') !== null) {
+        const receivedIngredients = localStorage.getItem('ingredients').split(',')
+        console.log(receivedIngredients)
+        receivedIngredients.forEach(ing => {
+            ingredientsSet.add(ing)
+            createIngredients()
+        })
+    }
+}
+
+export const updateIngredientsList = () => {
+    ingredientsSet.delete('')
+    ingredientsSet.delete(undefined)
```

Comment:
?

Reply:
I don't know why and how, but at the some moment these two above are creating in set